### PR TITLE
Tweaks to CI

### PR DESCRIPTION
**Pull request by omus**

### CODE REVIEW

**Comment by omus** [comment]
bors try

**Comment by fchorney** [comment]
At this point I think all the `1.3` stuff should be `1.5`? A little redundant since `1` is `1.5` right now, but will change to 1.6 soon enough right?

**Comment by fchorney** [comment]
Might also want to implement these cache changes https://github.com/invenia/Hyperparameters.jl/pull/8

**Comment by fchorney** [comment]
Also the slack notification if should be `always() && github.event_name == 'schedule'` instead of `github.event == 'schedule'`

**Comment by omus** [comment]
> At this point I think all the `1.3` stuff should be `1.5`? A little redundant since `1` is `1.5` right now, but will change to 1.6 soon enough right?

You're not wrong. I'm planning on backporting this commit to AWSS3 0.6 so I'd rather keep 1.3 testing enabled for that.

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
     name: Notify Slack Failure
     needs: test
     runs-on: ubuntu-latest
-    if: github.event == 'schedule'
+    if: always() && github.event == 'schedule'
```
Comment (fchorney) [inline .github/workflows/CI.yml:61]:
```suggestion
    if: always() && github.event_name == 'schedule'
```

Comment (omus) [reply .github/workflows/CI.yml:61]: That's what I get for being hasty

**Comment by omus** [comment]
bors try

**Comment by omus** [comment]
bors try

**Review by mattBrzezinski — APPROVED** [review]

**Comment by mattBrzezinski** [comment]
Just needed the magic touch ;)

**Pull request merged**